src/native/external/llvm-libunwind/include/__libunwind_config.h

```diff
@@ -178,17 +178,13 @@
 #if __loongarch_grlen == 64
 #define _LIBUNWIND_CONTEXT_SIZE 98
 #define _LIBUNWIND_CURSOR_SIZE 110
-#elif defined(HOST_WASM)
```

Comment:
@radekdoulik do we need this change or have your recent changes made it unnecessary?

Reply:
I moved it to line 187 where it was defined. Note that this was wrong because it was actually under #ifdef loongarch64 (see the error below: "Unsupported LoongArch ABI").

Reply:
Ah, thank you for the clarification, I haven't noticed that it was moved.